src/aspire/source/simulation.py

```diff
@@ -96,6 +97,41 @@ def __init__(
             logger.info("Appending a NoiseAdder to generation pipeline")
             self.noise_adder = NoiseAdder(seed=self.seed, noise_filter=noise_filter)
 
+    def _populate_ctf_metadata(self, filter_indices):
+        # Since we are not reading from a starfile, we must construct
+        # metadata based on the CTF filters by hand and set the values
+        # for these columns
+        #
+        # class attributes of CTFFilter:
+        CTFFilter_attributes = (
```

Comment:
Not sure if these would be better as a dictionary mapping between `CTFFilter_attributes` and `CTF_params`. They look similar. At least similar enough I had to look if exactly the same...

Reply:
Yeah there is a one-to-one correspondence. But the two sets of strings are used separately, so I think to do the `set_metadata` line that follows this line, I'd have to do something like `CTFDict.keys()`, which I would probably find more confusing if I were a newcomer. Unless I'm missing something